<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-13T15:16:55Z" name="Suite" finished-at="2021-09-13T15:17:39Z" duration-ms="44528">
    <groups>
    </groups>
    <test started-at="2021-09-13T15:16:55Z" name="Test" finished-at="2021-09-13T15:17:39Z" duration-ms="44528">
      <class name="superAdmin.BWS1_Login">
        <test-method is-config="true" signature="ExtentReport()[pri:0, instance:superAdmin.BWS1_Login@20c0a64d]" started-at="2021-09-13T15:16:55Z" name="ExtentReport" finished-at="2021-09-13T15:16:55Z" duration-ms="204" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentReport -->
        <test-method signature="Login()[pri:0, instance:superAdmin.BWS1_Login@20c0a64d]" started-at="2021-09-13T15:16:55Z" name="Login" finished-at="2021-09-13T15:17:17Z" duration-ms="21905" status="FAIL">
          <exception class="java.io.IOException">
            <message>
              <![CDATA[Destination 'F:\Brandscape\BrandscapeAutomation\SS' directory cannot be created]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: Destination 'F:\Brandscape\BrandscapeAutomation\SS' directory cannot be created
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1070)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1028)
at baseClass.BaseClass.TakeScreenshot(BaseClass.java:447)
at superAdmin.BWS1_Login.Login(BWS1_Login.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
      </class> <!-- superAdmin.BWS1_Login -->
      <class name="superAdmin.BWS903_Logout">
        <test-method signature="Logout()[pri:0, instance:superAdmin.BWS903_Logout@4ddbbdf8]" started-at="2021-09-13T15:17:30Z" name="Logout" finished-at="2021-09-13T15:17:39Z" duration-ms="8393" status="FAIL">
          <exception class="java.io.IOException">
            <message>
              <![CDATA[Destination 'F:\Brandscape\BrandscapeAutomation\SS' directory cannot be created]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: Destination 'F:\Brandscape\BrandscapeAutomation\SS' directory cannot be created
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1070)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1028)
at baseClass.BaseClass.TakeScreenshot(BaseClass.java:447)
at superAdmin.BWS903_Logout.Logout(BWS903_Logout.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Logout -->
        <test-method is-config="true" signature="ReportFlush()[pri:0, instance:superAdmin.BWS903_Logout@4ddbbdf8]" started-at="2021-09-13T15:17:39Z" name="ReportFlush" finished-at="2021-09-13T15:17:39Z" duration-ms="337" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ReportFlush -->
      </class> <!-- superAdmin.BWS903_Logout -->
      <class name="superAdmin.BWS2_AddAdmin">
        <test-method signature="AddAdmin()[pri:0, instance:superAdmin.BWS2_AddAdmin@455b6df1]" started-at="2021-09-13T15:17:17Z" name="AddAdmin" finished-at="2021-09-13T15:17:30Z" duration-ms="13663" status="FAIL">
          <exception class="java.io.IOException">
            <message>
              <![CDATA[Destination 'F:\Brandscape\BrandscapeAutomation\SS' directory cannot be created]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: Destination 'F:\Brandscape\BrandscapeAutomation\SS' directory cannot be created
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1070)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1028)
at baseClass.BaseClass.TakeScreenshot(BaseClass.java:447)
at superAdmin.BWS2_AddAdmin.AddAdmin(BWS2_AddAdmin.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddAdmin -->
      </class> <!-- superAdmin.BWS2_AddAdmin -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
